openapi.json

```diff
@@ -480,7 +480,7 @@
         }
       },
       "AddToConference": {
-        "description": "The `AddToConference` command adds a Participant to a Conference. If this Participant currently is in another Conference, the Participant is first removed from that Conference. Two Call legs can be bridged together by creating a Conference and adding both Call legs to it via `AddToConference`.",
+        "description": "The `AddToConference` command adds a Participant to a Conference. Two Call legs can be bridged together by creating a Conference and adding both Call legs to it via `AddToConference`.",
```

Comment:
@amahadaya - i thought we added test changes in the version that was created using the VCSWP-22711 apiserver spec branch?

Reply:
There was a main branch change that changed the addToConference description that we needed to include, this is a part of the latest main branch, VCSWP-22711 was approved and merged to main